src/commands/kv/namespace/site.rs

```diff
@@ -0,0 +1,60 @@
+use cloudflare::endpoints::workerskv::create_namespace::CreateNamespace;
+use cloudflare::endpoints::workerskv::create_namespace::CreateNamespaceParams;
+use cloudflare::endpoints::workerskv::list_namespaces::ListNamespaces;
+use cloudflare::endpoints::workerskv::WorkersKvNamespace;
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::response::ApiFailure;
+
+use crate::commands::kv;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::target::Target;
+use crate::terminal::message;
+
+pub fn site(target: &Target, user: &GlobalUser) -> Result<WorkersKvNamespace, failure::Error> {
+    let client = kv::api_client(user.to_owned())?;
+
+    let title = format!("__{}-{}", target.name, "workers_sites_assets");
+    let msg = format!("Creating namespace for Workers Site \"{}\"", title);
+    message::working(&msg);
+
+    let response = client.request(&CreateNamespace {
+        account_identifier: &target.account_id,
+        params: CreateNamespaceParams {
+            title: title.to_owned(),
+        },
+    });
+
+    match response {
+        Ok(success) => {
+            return Ok(success.result);
+        }
+        Err(e) => match e {
+            ApiFailure::Error(_status, api_errors) => {
+                if api_errors
+                    .errors
+                    .iter()
+                    .find(|&e| e.code == 10014)
+                    .is_some()
+                {
+                    log::info!("Namespace {} already exists.", title);
+                    let response = client.request(&ListNamespaces {
```

Comment:
I think it's worth refactoring list() and create() so that the API call happens in a separate (private) function, then two public functions call to it (the default `kv:namespace` one, and then this function for sites!)

Reply:
i'd be stoked to make refactors later.

Reply:
i think it is too; i'm happy to do this later this week or early next week once we've cut a release candidate.